src/ProxyClient/MultiplexerClient.php

```diff
@@ -131,13 +165,19 @@ public function refresh($url, array $headers = [])
      * @param string $interface The FQN of the interface
      * @param string $method    The method to invoke
      * @param array  $arguments The arguments to be passed to the method
+     *
+     * @return mixed Return value of first method call
      */
     private function invoke($interface, $method, array $arguments)
     {
+        $result = null;
+
         foreach ($this->proxyClients as $proxyClient) {
             if (is_subclass_of($proxyClient, $interface)) {
-                call_user_func_array([$proxyClient, $method], $arguments);
+                $result = call_user_func_array([$proxyClient, $method], $arguments);
             }
         }
+
+        return $result;
```

Comment:
We have to return a string for header name and value (not an array), so I’m simply returning the value of the last TagCapable client. Does that make sense? Or should we just return null or some constant string `multiple`?